src/lib/dom_helpers.js

```diff
@@ -32,7 +43,7 @@ function bindFocusables( instance, activateOnFocus ) {
             if ( onFocusPrev ) onFocusPrev.call( element, event );
           };
         };
-        for ( let element of [ ...focusables ] ) {
+        for ( let element of nodeListToArray(focusables) ) {
           element.onfocus = onFocus( element );
         }
```

Comment:
Thank you for this. What do you think about taking this approach instead? 
```
Array.prototype.slice.call( focusables ).forEach( element => ...
```
This should be simpler and produce less code on build.